File: audioplay/app/src/main/java/com/example/audioplay/MainActivity.java
package com.example.audioplay;

import androidx.appcompat.app.AppCompatActivity;
import android.media.MediaPlayer;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    MediaPlayer mp;
    Button play, pause;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Initialize MediaPlayer and Buttons
        mp = MediaPlayer.create(MainActivity.this, R.raw.song);
        play = findViewById(R.id.playId);
        pause = findViewById(R.id.pauseId);

        // Set OnClickListener for play button
        play.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                mp.start(); // Start playing audio
                Toast.makeText(MainActivity.this, "Playing Audio", Toast.LENGTH_SHORT).show();
            }
        });

        // Set OnClickListener for pause button
        pause.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                mp.pause(); // Pause audio
                pause.setEnabled(false); // Disable pause button
                mp.release(); // Release resources used by MediaPlayer
                Toast.makeText(getApplicationContext(), "Pausing Audio", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: kotlinsample/app/src/main/java/com/example/kotlinsample/MainActivity.kt
package com.example.kotlinsample

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // Find views by their IDs
        val showButton = findViewById<Button>(R.id.showInput)
        val editText = findViewById<EditText>(R.id.editText)

        // Set a click listener on the button
        showButton.setOnClickListener {
            // Display the text entered in the EditText as a Toast message
            Toast.makeText(this, editText.text, Toast.LENGTH_SHORT).show()
        }
    }
}
